Formula/cnosdb.rb

```diff
@@ -0,0 +1,50 @@
+class Cnosdb < Formula
+  desc "Open source distributed time series database"
+  homepage "https://www.cnosdb.com"
+  url "https://github.com/cnosdb/cnosdb.git",
+      tag:      "v1.0.2",
+      revision: "bf128f15334a9104a5a644917693c29bf5eb3e65"
+  license "MIT"
+  head "https://github.com/cnosdb/cnosdb.git", branch: "main"
+  # The regex below omits a rogue `v9.9.9` tag that breaks version comparison.
+  livecheck do
+    url :stable
+    regex(/^v?((?!9\.9\.9)\d+(?:\.\d+)+)$/i)
+  end
+  depends_on "go" => :build
+  def install
+    ldflags = %W[
+      -s
+      -w
+      -X main.version=#{version}
+      -X main.commit=#{Utils.git_short_head(length: 10)}
+      -X main.date=#{time.iso8601}
+    ]
+    system "go", "build", *std_go_args(output: bin/"cnosdb", ldflags: ldflags), "./cmd/cnosdb"
+    system "go", "build", *std_go_args(output: bin/"cnosdb-cli", ldflags: ldflags), "./cmd/cnosdb-cli"
+    system "go", "build", *std_go_args(output: bin/"cnosdb-inspect", ldflags: ldflags), "./cmd/cnosdb-inspect"
+    system "go", "build", *std_go_args(output: bin/"cnosdb-tools", ldflags: ldflags), "./cmd/cnosdb-tools"
+    system "go", "build", *std_go_args(output: bin/"cnosdb-ctl", ldflags: ldflags), "./cmd/cnosdb-ctl"
+    system "go", "build", *std_go_args(output: bin/"cnosdb-meta", ldflags: ldflags), "./cmd/cnosdb-meta"
+  end
+  test do
+    cnosdb_port = free_port
+    (testpath/"config.toml").write shell_output("#{bin}/cnosdb config")
+    inreplace testpath/"config.toml" do |s|
```

Comment:
I think this is missing the port setting

Reply:
yes, I will fix it. Thank you.